### ISSUE TITLE
cinnamon: Startup sound effect is poor quality

### ISSUE THREAD

**Issue by tymbase** [linuxmint/mint22.1-beta#65]
Hi dont know if minor issue has been noticed but has gone on since 20.3,part of the startup chime is missing still same on latest beta.P.S. using LMDE6 best ever great work rock stable.

**Comment by Reno-Sifana** [comment]
> Hi dont know if minor issue has been noticed but has gone on since 20.3,part of the startup chime is missing still same on latest beta.P.S. using LMDE6 best ever great work rock stable.

i've this problem too. since upgrading Linux Mint 21.x series. but so strange, because when I'm plugged into my Headset the startup chime is heard. and it's so odd. maybe @clefebvre can explain this.
Notes: Sorry if my English is bad.

**Comment by DirkS2009** [comment]
Seems to be individually. On my PC and Notebook the startup sound is there on LM22 and LM22.1 beta. Both fresh installations.

**Comment by tymbase** [comment]
Hi everyone thanks for noticing ,it seems the login oga file has somehow been corrupted,it is easy to overcome however by replacing the file by one from an earlier Mint version.

**Comment by mtwebster** [comment]
> when I'm plugged into my Headset the startup chime is heard. and it's so odd.

Where are the speakers used if your headphones aren't used? Speakers integrated in a monitor can be affected by video drivers (nvidia power-saving would frequently cause my speakers to need 'woken up', and delay sounds).

**Comment by Reno-Sifana** [comment]
> > when I'm plugged into my Headset the startup chime is heard. and it's so odd.
> 
> Where are the speakers used if your headphones aren't used? Speakers integrated in a monitor can be affected by video drivers (nvidia power-saving would frequently cause my speakers to need 'woken up', and delay sounds).
> 
No. I'm not using monitor but an old laptop Toshiba Satellite L645 powered Intel Core I3 350M (and of course no Nvidia graphic card). But the strange is when I click shut down / logout sound is played correctly on laptop Speaker / Gaming Headset.

**Comment by tymbase** [comment]
Hi everybody thanks for noticing,I think it likely it is just the file itself. I have checked properties on the current file 55.1kB ,file from 19.3, 99.2kB seems about half of it has been lost.All other sounds work OK. Worried quality control has slipped, having been a long time user and enthusiast for the best Linux distro in the world which I have used for many years as my daily driver.Cheers everybody,best wishes to Clem and all at Mint.

**Comment by mtwebster** [comment]
The 55.1kb file is the correct file and is not corrupted.

The login sound (and others) was changed for Mint 21.1:
https://github.com/linuxmint/mint-artwork/commit/7befea5cdf8ef10bbf244110a3ba594d85199903

**Comment by tymbase** [comment]
mtwebster Have you actually listened to the login sound ,what is it supposed to sound like? To me as an audio engineer it is just like a tape which has been cut only the latter half can be heard.Ill go away now and leave you in peace,only to say there are none so deaf as those who dont listen.

**Comment by mtwebster** [comment]
I don't know how this new sound was constructed. Possibly it is exactly a trimmed part of the original, deliberately done, because he liked it better (I do also, the old one was too flamboyant for my taste). Unfortunately, we are *not* audio engineers. Sometimes it's just a best effort - when I need a new icon I'll (crudely) modify or mash up existing ones. Eventually a nice artist will come along and set things right for me.

I closed the issue because it seemed based on confusion - some unexpected corruption in the iso files.

**Comment by clefebvre** [comment]
I'm happy to welcome PRs and consider better sounds, especially from sound engineers.

In the meantime though, I don't see a bug here.
